sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_client.py

```diff
@@ -773,6 +815,7 @@ def get_random_bytes(self, count: int, **kwargs) -> bytes:
 
         :return: The random bytes.
         :rtype: bytes
+
```

Comment:
Empty line?

Reply:
This is actually intentional, to separate the method return information from the raising behavior documentation. It's not necessary, but aligns with what we do throughout the library.